src/main/java/core/basesyntax/model/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Rectangle extends Color {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double width, double length) {
+        super(color);
+        this.width = width;
+        this.length = length;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq. units, length: "
```

Comment:
Magic numbers like 'sq. units' and 'units' should be defined as constants to improve code readability and maintainability.